test/test_jit.py

```diff
@@ -6323,6 +6323,126 @@ def test_script_module_export_resnet18(self):
         self.assertEqual(output_orig, output_import)
         self.assertEqual(grad_orig, grad_import)
 
+    @skipIfNoTorchVision
```

Comment:
Why do you even require torch vision if you're implementing ResNet by yourself?? It would really be much better to use the one from torch vision...

Reply:
Aha this is because I don't want to enable this test in normal CI test runs (it increase the test time for test_jit.py at least 15s). This implementation is basically copied from torch vision, with TorchScript annotation to test all relative modules. To use the one from torch vision, it require us to annotate torch vision.py with weak script mode, which might not be a good idea because we don't know which pytorch version the user installed for pytorch/vision users.

Reply:
^--- what you wrote above, put it in a comment on the code

Reply:
Also, what’s the purpose of having a test like it if it’s disabled... it only adds 15s to all local test runs I do, and doesn’t actually make our codebase safer...

Reply:
it still runs under some CI build which have the torch/vision installed, so some part of CI actually do check this test (as well as other `SkipIfNoTorchVision` tests). If we enable this test(probably also the other resnet_trace test that is annotated as `SkipIfNoTorchVision`) for all CI builds, we will add up about 30s for all CI builds that runs test_jit. I am not sure if that's a good idea or not, if it is ok for us, I can actually enable them :)